cfg.sample.toml

```diff
@@ -29,6 +29,9 @@ name = ""
 # who has r+ rights?
 reviewers = ["barosl", "graydon"]
 
+# who has 'try' rights? (try, retry, force, clean, prioritization)
+try = []
```

Comment:
Hmm. At first I thought this `try` configuration was for merely initiating try builds, but it seems that the reviewers in this category has more power than that. Probably something like `maintainers` would be better? If this sounds too vague, I'm OK with `try_reviewers`.